.github/workflows/ci.yaml

```
@@ -15,47 +15,45 @@ permissions:
     contents: write # needed for mutex
 
 jobs:
+  get-values:
+    uses: ./.github/workflows/get-values.yaml
+    permissions:
+      contents: write # needed updating dependabot branches
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)